obs_status.yml

```diff
@@ -0,0 +1,2147 @@
+  status: 1
+- agasc_id:
+  - 857375848
+  comments: Expected star position is >1000 arcsec away during the whole obs
+  mp_starcat_time: 2003:357:02:49:14.852
+  status: 1
+- agasc_id:
+  - 188482256
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2004:055:07:41:03.254
+  status: 1
+- agasc_id:
+  - 654050016
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2004:109:04:09:26.608
+  status: 1
+- agasc_id:
+  - 920782072
+  comments: NSM at start of observation
+  mp_starcat_time: 2004:200:06:56:32.527
+  status: 1
+- agasc_id:
+  - 920783352
+  comments: NSM at start of observation
+  mp_starcat_time: 2004:200:06:56:32.527
+  status: 1
+- agasc_id:
+  - 897607632
+  comments: Large magnitude variation. Known eclipsing binary
+  mp_starcat_time: 2004:212:17:56:19.266
+  status: 1
+- agasc_id:
+  - 76813072
+  comments: mag 8.1 spoiler 15 arcsec away
+  mp_starcat_time: 2004:277:04:10:18.167
+  status: 1
+- agasc_id:
+  - 257953336
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2005:038:01:15:06.000
+  status: 1
+- agasc_id:
+  - 35263288
+  comments: Faint star with imposter. Magnitude is good
+  mp_starcat_time: 2005:056:20:11:01.041
+  status: 0
+- agasc_id:
+  - 723657296
+  comments: Faint star. Almost never tracked. Set mag to 11.547
+  mp_starcat_time: 2005:074:22:44:33.953
+  status: 1
+- agasc_id:
+  - 652480464
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2005:150:01:28:35.423
+  status: 1
+- agasc_id:
+  - 615914552
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2005:281:04:44:20.882
+  status: 1
+- agasc_id:
+  - 839385704
+  comments: Almost never acquired. When acquired, it was too bright. Star has been
+    observed other times. Known symbiotic star
+  mp_starcat_time: 2005:282:18:53:05.394
+  status: 1
+- agasc_id:
+  - 1197635184
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2005:353:06:48:26.153
+  status: 1
+- agasc_id:
+  - 1197635184
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2005:354:14:42:59.384
+  status: 1
+- agasc_id:
+  - 1197635184
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2005:355:23:10:36.631
+  status: 1
+- agasc_id:
+  - 118498608
+  comments: Faint star. Affected by imposters. Set mag to MAXMAG
+  mp_starcat_time: 2006:014:13:25:39.795
+  status: 1
+- agasc_id:
+  - 330967424
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2006:064:07:51:01.152
+  status: 1
+- agasc_id:
+  - 758255832
+  comments: Venus observation
+  mp_starcat_time: 2006:086:03:56:54.298
+  status: 1
+- agasc_id:
+  - 758255832
+  comments: Venus observation
+  mp_starcat_time: 2006:086:06:10:03.448
+  status: 1
+- agasc_id:
+  - 758255832
+  comments: Venus observation
+  mp_starcat_time: 2006:086:08:01:53.448
+  status: 1
+- agasc_id:
+  - 759958456
+  comments: Venus observation
+  mp_starcat_time: 2006:086:08:01:53.448
```

Comment:
I thought we were going to exclude all of the Venus star observations?  If so, I think this needs entries for at least 2006:086:11:45:33.448 and 2006:086:13:37:23.448 too?

Reply:
What happens is that not all Venus star-observations end up being flagged by the process. Instead of adding star-observations separately, I intend to add the observations themselves, which in turn will remove all star-observations for those OBSIDs.

Reply:
Understood and that's what I figured.  It just means that there are Venus star-obs combinations now with accompanying mag information in the current released supplement and the update to add all the Venus star-observations themselves will strip those out of the table.  I *think* that is fine; I just hadn't realized they were already in the data.